data/transport missions.txt

```diff
@@ -1412,6 +1412,137 @@ mission "Hallucination"
 
 
 
+mission "Shady passenger transport 1"
+	name "Shady passenger transport"
+	description "Discreetly transport a passenger to <destination>. If you are caught with him, you may be fined."
+	minor
+	repeat
+	source
+		attributes "near earth" core urban
+	destination
+		attributes frontier "dirt belt" rim north south pirate
+		distance 2 300
```

Comment:
`300`? transport 2 has `7` as the max.

Reply:
Oops, those two zeroes snuck in there by mistake.